test/variable.jl

```diff
@@ -1077,6 +1077,80 @@ function test_Model_VariableIndex_VariableRef_fix_with_upper_bound(::Any, ::Any)
     return
 end
 
+function test_Hermitian_PSD(::Any, ::Any)
+    model = Model()
+    @variable(model, Q[1:2, 1:2] in HermitianPSDCone())
+    @test num_variables(model) == 4
+    v = all_variables(model)
+    _test_variable_name_util(v[1], "real(Q[1,1])")
+    @test Q[1, 1] == 1v[1]
+    _test_variable_name_util(v[2], "real(Q[1,2])")
+    _test_variable_name_util(v[4], "imag(Q[1,2])")
+    @test Q[1, 2] == v[2] + v[4] * im
+    _test_variable_name_util(v[3], "real(Q[2,2])")
+    @test Q[2, 2] == 1v[3]
+    @test Q[2, 1] == conj(Q[1, 2])
+end
+
+function test_Hermitian_PSD_errors(ModelType, ::Any)
+    model = ModelType()
+    @test_throws ErrorException @variable(
+        model,
+        H[i = 1:2, j = 1:2] in HermitianPSDCone(),
+        lower_bound = (i + j) * im
+    )
+    @test_throws ErrorException @variable(
+        model,
+        H[i = 1:2, j = 1:2] in HermitianPSDCone(),
+        upper_bound = (i + j) * im
+    )
+    @test_throws ErrorException @variable(
+        model,
+        H[i = 1:2, j = 1:2] in HermitianPSDCone(),
+        start = (i + j) * im
+    )
+    @test_throws ErrorException @variable(
+        model,
+        H[i = 1:2, j = 1:2] in HermitianPSDCone(),
+        integer = i > j
+    )
+    @test_throws ErrorException @variable(
+        model,
+        H[i = 1:2, j = 1:2] in HermitianPSDCone(),
+        Bool
```

Comment:
```suggestion
        Bin
```

Not sure what `Bool` was meant to do here?

Reply:
Oops